package.json

```diff
@@ -1,30 +1,36 @@
 {
-	"name": "testingserver",
-	"version": "1.0.0",
-	"description": "Server Testing Project",
-	"main": "server.js",
-	"scripts": {
-		"test": "jest --watch --verbose"
-	},
-	"jest": {
-		"testEnvironment": "node"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/LambdaSchool/Server-Testing.git"
-	},
-	"keywords": [],
-	"author": "",
-	"license": "ISC",
-	"bugs": {
-		"url": "https://github.com/LambdaSchool/Server-Testing/issues"
-	},
-	"homepage": "https://github.com/LambdaSchool/Server-Testing#readme",
-	"devDependencies": {
-		"jest": "^23.6.0",
-		"supertest": "^3.3.0"
-	},
-	"dependencies": {
-		"express": "^4.16.4"
-	}
+  "name": "testingserver",
+  "version": "1.0.0",
+  "description": "Server Testing Project",
+  "main": "server.js",
+  "scripts": {
+    "test": "jest --watch --verbose",
+    "start": "node index.js",
+    "server": "nodemon index.js"
+  },
+  "jest": {
+    "testEnvironment": "node"
+  },
+  "repository": "git+https://github.com/LambdaSchool/Server-Testing.git",
+  "keywords": [],
+  "author": "VictorGoic0 <vgoico000@citymail.cuny.edu>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/LambdaSchool/Server-Testing/issues"
+  },
+  "homepage": "https://github.com/LambdaSchool/Server-Testing#readme",
+  "devDependencies": {
+    "cross-env": "^5.2.0",
```

Comment:
ah, so you did bring this in but never include it in your test script--does this mean your local .env file contains DB_ENV=testing? that's alright if it does for the purpose of the project, just remember that you want control over all three main run environments, dev, test, production (deployment), if you use cross-env in your `test` script the way Luis had it yesterday and your DB_ENV is set to "development" in your .env file, the test script command will override it and will assign that variable as "testing"